projects/schematics/package.json

```diff
@@ -9,7 +9,8 @@
     "schematics"
   ],
   "scripts": {
-    "clean": "rimraf \"src/**/*.js\" \"src/**/*.js.map\" \"src/**/*.d.ts\"",
+    "prepare:schematics": "yarn install",
```

Comment:
is this really needed?

Reply:
Yes, when the CI is running it's needed.